File: app/src/main/java/com/example/testapp/PaymentRequest.java
package com.example.testapp;

import android.content.Intent;
import android.graphics.Color;
import android.os.Bundle;
import android.view.View;
import android.view.ViewGroup;
import android.widget.AdapterView;
import android.widget.ArrayAdapter;
import android.widget.AutoCompleteTextView;
import android.widget.Button;
import android.widget.Spinner;
import android.widget.TextView;
import android.widget.Toast;

import androidx.appcompat.app.AppCompatActivity;

import java.util.ArrayList;
import java.util.List;

public class PaymentRequest extends AppCompatActivity {
    private Spinner spinner1;
    private Spinner spinner2;
    private Button submit;
    private AutoCompleteTextView autoCompleteTextView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_payment_request);

        submit = findViewById(R.id.submit);
        submit.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Toast.makeText(PaymentRequest.this, "Payment Submit Successfully✔✔", Toast.LENGTH_SHORT).show();
                Intent intent = new Intent(PaymentRequest.this, MainActivity.class);
                startActivity(intent);
            }
        });

        ArrayList<String> spinner1Items = new ArrayList<>();
        spinner1Items.add("Please Select"); spinner1Items.add("Type 1"); spinner1Items.add("Type  2") ;
        spinner1 = findViewById(R.id.spinner3);

        final ArrayAdapter<String> spinner1Adapter = new ArrayAdapter<String>(
                this,android.R.layout.simple_spinner_item,  spinner1Items ){
            @Override
            public boolean isEnabled(int position){
                if(position == 0)
                {
                    return false;
                }
                else
                {
                    return true;
                }
            }
            @Override
            public View getDropDownView(int position, View convertView,
                                        ViewGroup parent) {
                View view = super.getDropDownView(position, convertView, parent);
                TextView tv = (TextView) view;
                if(position == 0){

                    tv.setTextColor(Color.GRAY);
                }
                else {
                    tv.setTextColor(Color.BLACK);
                }
                return view;
            }
        };
        spinner1Adapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
        spinner1.setAdapter(spinner1Adapter);

        ArrayList<String> spinner2Items = new ArrayList<>();
        spinner2Items.add("Please Select"); spinner2Items.add("Type 1"); spinner2Items.add("Type 2") ;
        spinner2 = findViewById(R.id.spinner4);
        final ArrayAdapter<String> spinner2Adapter = new ArrayAdapter<String>(
                this,android.R.layout.simple_spinner_item,  spinner2Items ){
            @Override
            public boolean isEnabled(int position){
                if(position == 0)
                {
                    return false;
                }
                else
                {
                    return true;
                }
            }
            @Override
            public View getDropDownView(int position, View convertView,
                                        ViewGroup parent) {
                View view = super.getDropDownView(position, convertView, parent);
                TextView tv = (TextView) view;
                if(position == 0){

                    tv.setTextColor(Color.GRAY);
                }
                else {
                    tv.setTextColor(Color.BLACK);
                }
                return view;
            }
        };
        spinner2Adapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
        spinner2.setAdapter(spinner2Adapter);


    }
}

File: app/src/main/java/com/example/testapp/PaymentRequest.kt
package com.example.testapp

import android.annotation.SuppressLint
import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.AdapterView
import android.widget.ArrayAdapter
import android.widget.AutoCompleteTextView
import android.widget.Button
import android.widget.Spinner
import android.widget.Toast
import androidx.core.content.ContentProviderCompat.requireContext

class PaymentRequest : AppCompatActivity() {
    private lateinit var spinner1 : Spinner
    private lateinit var spinner2 : Spinner
    private lateinit var submit : Button
    private lateinit var autotextView : AutoCompleteTextView
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_payment_request)
//        autotextView = findViewById(R.id.autotextView);
        submit = findViewById(R.id.submit)
        submit.setOnClickListener{
            Toast.makeText(this@PaymentRequest , "Payment Submit Successfully✔✔" , Toast.LENGTH_SHORT).show()
            val intent = Intent( this@PaymentRequest , MainActivity::class.java)
            startActivity(intent)
        }

//        val userArr = ArrayList<String>();
//        for(i in 1..100){
//            userArr.add("User $i")
//        }

//        val adapter = ArrayAdapter(this,
//            android.R.layout.simple_list_item_1, userArr)
//        autotextView.setAdapter(adapter)


//        LedgerName.setOnClickListener(View.OnClickListener {
//            if (linerLayout1.visibility != View.VISIBLE) {
//                linerLayout1.visibility = View.VISIBLE
//                autotextView.setEnabled(true)
//                autotextView.onItemClickListener =
//                    AdapterView.OnItemClickListener { _, _, _, _ ->
//                        val name: String = autotextView.text.toString()
//                        LedgerName.text = name
//                        autotextView.setText("")
//                        autotextView.isEnabled = false
//                        linerLayout1.visibility = View.GONE
//                    }
//            } else {
//                linerLayout1.visibility = View.GONE
//            }
//        })


        val spinner1Items = listOf("Test User 1", "Test User 2")
        val spinner1Adapter = ArrayAdapter(this, android.R.layout.simple_spinner_item, spinner1Items)
        spinner1Adapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item)
        spinner1 = findViewById(R.id.spinner3)
        spinner1.adapter = spinner1Adapter
        spinner1.onItemSelectedListener = object : AdapterView.OnItemSelectedListener {
            override fun onItemSelected(parent: AdapterView<*>?, view: View?, position: Int, id: Long) {
            }
            override fun onNothingSelected(parent: AdapterView<*>?) {
            }
        }
        val spinner2Items = listOf("Test Sub User 1", "Test Sub User 2")
        val spinner2Adapter = ArrayAdapter(this, android.R.layout.simple_spinner_item, spinner2Items)
        spinner2Adapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item)
        spinner2 = findViewById(R.id.spinner4)
        spinner2.adapter = spinner2Adapter
        spinner2.onItemSelectedListener = object : AdapterView.OnItemSelectedListener {
            override fun onItemSelected(parent: AdapterView<*>?, view: View?, position: Int, id: Long) {}
            override fun onNothingSelected(parent: AdapterView<*>?) {}
        }
    }


}